cpp/cmake_modules/DefineOptions.cmake

```diff
@@ -264,7 +264,8 @@ takes precedence over ccache if a storage backend is configured" ON)
                 "Build Arrow Fuzzing executables"
                 OFF
                 DEPENDS
-                ARROW_TESTING)
+                ARROW_TESTING
+                ARROW_WITH_BROTLI)
```

Comment:
This is nice, but it means build scripts will error out when options don't satisfy these dependencies, right?

Reply:
It's not right.
It means that "-DARROW_WITH_BROTLI=ON is added implicitly when -DARROW_FUZZING=ON is specified".
We don't need to update the CI scripts .